### PR TITLE
Add conversion for snowflake ids from POSIXct

### PR DESCRIPTION
**Pull request by Kudusch**
See #153 

This makes it possible to supply a POSIXct object to max_id, min_id and since_id to enable easier search/query using dates

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -237,15 +237,23 @@ rtoot_ask <- function(prompt = "enter authorization code: ", pass = TRUE, check_
     return(passFun(prompt = prompt))
 }
 
+handle_id <- function(x) {
+    ## Convert x to snowflake id if it is POSIXct
+    if (is (x, "POSIXct")) {
+        return(as.numeric(x)*(2^16))*1000)
```
Comment (chainsawriot) [inline R/utils.R:243]: Sorry, my bad:

```r
handle_id <- function(x) {
    ## Convert x to snowflake id if it is POSIXct
    if (is(x, "POSIXct")) {
        return(as.numeric(x) * (2^16) * 1000)
    }
    x
}
```

It is also worth mentioning that there are bitshift operators in R.

```r
bitwShiftL(10, 16) == (10 * 2^16)
```

**Review by chainsawriot — CHANGES_REQUESTED** [review]
Please also add yourself to `DESCRIPTION`.

File: R/utils.R (hunk 1 above)
Comment (Kudusch) [reply R/utils.R:243]: I had trouble getting bitwShiftL to work because the mastodon ids are bigger then the max integer (at least in my testing).

Comment (chainsawriot) [reply R/utils.R:243]: Ah, yes, 64-bit integer.

**Review by chainsawriot — APPROVED** [review]

**Pull request merged**